stable/rabbitmq/README.md

```diff
@@ -0,0 +1,88 @@
+# RabbitMQ
+
+[RabbitMQ](https://www.rabbitmq.com/) is an open source message broker software that implements the Advanced Message Queuing Protocol (AMQP).
+
+## TL;DR;
+
+```bash
+$ helm install stable/rabbitmq
+```
+
+## Introduction
+
+This chart bootstraps a [RabbitMQ](https://github.com/bitnami/bitnami-docker-rabbitmq) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/rabbitmq
+```
+
+The command deploys RabbitMQ on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the RabbitMQ chart and their default values.
+
+|         Parameter          |                       Description                       |                         Default                          |
+|----------------------------|---------------------------------------------------------|----------------------------------------------------------|
+| `image`                    | RabbitMQ image                                          | `bitnami/rabbitmq:{VERSION}`                             |
+| `imagePullPolicy`          | Image pull policy                                       | `Always` if `imageTag` is `latest`, else `IfNotPresent`. |
+| `rabbitmqUsername`         | RabbitMQ application username                           | `user`                                                   |
+| `rabbitmqPassword`         | RabbitMQ application password                           | `bitnami`                                                |
```

Comment:
done, additionally the erlang cookie is also randomly generated when not specified by the user.